### ISSUE TITLE
TinyMCE in a dialog

### ISSUE THREAD

**Issue by jcgueriaud1** [parttio/tinymce-for-flow#39]
This code:
```
                   // Move aux element as child from body to element to fix menus in modal Dialog
                    const aux = document.getElementsByClassName('tox-tinymce-aux')[0];
                    aux.parentElement.removeChild(aux);
                    // Fix to allow menu grow outside Dialog
                    aux.style.position = 'absolute';
                    c.appendChild(aux);
```
To move the overlay inside the dialog is not working if you have multiple tinymce (then multiple 'tox-tinymce-aux').

Step to reproduce:
- open this view
- open the dialog
- Test tinymce in the dialog and in the view they are broken

Maybe one workaround is to apply the code everytime. And also only element from the children's body (to not steal the `tox-tinymce-aux` from another editor.
`const aux = document.querySelectorAll("body > div.tox-tinymce-aux")[0] as HTMLElement;`

```

@Route
public class EditorInDialog extends Div {

    public EditorInDialog() {
        Dialog dialog = new Dialog();
//        dialog.setModal(false);

        TinyMce editor = new TinyMce();
        editor.configure("plugins", "link");

        editor.addValueChangeListener(e -> {
            reportValue(e.getValue());
        });

        dialog.add(new Div(editor, new Button("show value", e -> {
            reportValue(editor.getValue());
        }), new Button("close", e -> {
            dialog.close();
        })));
        Button button = new Button("Open the editor dialog");
        button.addClickListener(event -> {
            dialog.open();
        });

        TinyMce editor2 = new TinyMce();
        add(editor2, button);

    }

    private void reportValue(String value) {
        // Not using notifications here because of regression in V24:
        // https://github.com/vaadin/flow-components/issues/4896
        add(new Paragraph(("Value now:" + value)));
    }

}

```

**Comment by TatuLund** [comment]
Fixed by https://github.com/parttio/tinymce-for-flow/commit/ea1a2d1ecc18dc7a3d041e72cccc5bb5665f2062

**Comment by TatuLund** [comment]
Fixed in version 4.2.5

**Comment by mstahv** [comment]
Closing as apparently it is fixed :-)
